consensus/state.go

```diff
@@ -729,7 +766,8 @@ func (cs *ConsensusState) EnterPrevoteWait(height int, round int) {
 
 	// After `timeoutPrevote0+timeoutPrevoteDelta*round`, EnterPrecommit()
 	go func() {
-		time.Sleep(timeoutPrevote0 + timeoutPrevote0*time.Duration(round))
+		time.Sleep(timeoutPrevote0 + timeoutPrevoteDelta*time.Duration(round))
+		cs.evsw.FireEvent(types.EventStringTimeoutWait(), cs.RoundStateEvent())
```

Comment:
Should move into EnterPrecommit with a boolean timeout=true, and fire event after conditional check.